### PR TITLE
update to 2024.3.11

### PR DESCRIPTION
**Pull request by cbouss**
Changes:
- update to 2024.3.11

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -15,6 +15,7 @@ source:
 
 build:
   number: 0
+  noarch: generic
```
Comment (JeanChristopheMorinPerso) [inline recipe/meta.yaml:18]: This won't work AFAIK unless we change the activation scripts in OpenSSL.

Comment (JeanChristopheMorinPerso) [reply recipe/meta.yaml:18]: Or more, it will work, but it will break stuff that rely on ca-certificates.

Comment (cbouss) [reply recipe/meta.yaml:18]: Oh that was unintended leftover from past local changes. Thanks for the catch.

**Review by JeanChristopheMorinPerso — APPROVED** [review]

**Review by JamesRobertsonGames — APPROVED** [review]
Looks good to me 👍

**Pull request merged**